src/js/index.js

```diff
@@ -9,7 +9,7 @@ import "bootstrap";
 import "../styles/index.scss";
 
 //import your own components
-import Home from "./component/home.jsx";
+import Home from "./component/Home.jsx";
 
 //render your react application
```

Comment:
Why have you commented this code?
Please remove all the comments.

Reply:
I haved removed the unnecesary comments along the files.